base/strings/string.jl

```diff
@@ -425,6 +425,17 @@ function reverse(s::String)
     String(buf)
 end
 
+"""
+    repeat(s::String, r::Integer)
+
+Repeat a string `r` times.
```

Comment:
the multiple docstrings would need to be distinguished by signature otherwise they will all be repeated in both sections